tests/test_locale.py

```diff
@@ -55,44 +55,3 @@ def test_add_message_catalog(app, rootdir):
     assert _('Hello world') == 'HELLO WORLD'
     assert _('Hello sphinx') == 'Hello sphinx'
     assert _('Hello reST') == 'Hello reST'
-
-
-def _empty_language_translation(rootdir):
-    locale_dirs, catalog = [rootdir / 'test-locale' / 'locale1'], 'myext'
-    locale.translators.clear()
-    locale.init(locale_dirs, language=None, catalog=catalog)
-    return locale.get_translation(catalog)
-
-
-def test_init_environment_language(rootdir, monkeypatch):
-    with monkeypatch.context() as m:
-        m.setenv("LANGUAGE", "en_US:en")
-        _ = _empty_language_translation(rootdir)
-        assert _('Hello world') == 'HELLO WORLD'
-
-    with monkeypatch.context() as m:
-        m.setenv("LANGUAGE", "et_EE:et")
-        _ = _empty_language_translation(rootdir)
-        assert _('Hello world') == 'Tere maailm'
```

Comment:
This test case may also be worth salvaging separately, because it doesn't rely on `SOURCE_DATE_EPOCH`.

Reply:
Kept!